src/mastercard-auth.js

```diff
@@ -16,8 +16,23 @@ module.exports = function (context) {
 
   const mastercard = context.request.getEnvironmentVariable('mastercard');
 
-  if (mastercard && commaDecodedUrl.includes('mastercard.com')) {
+  if(mastercard){
     try {
+      if(mastercard.appliesTo){
+        var validConfiguration = false
```

Comment:
I would maybe use `signRequest` or `interceptRequest` instead of `validConfiguration`